core/controllers/collection_editor_test.py

```diff
@@ -96,3 +97,19 @@ def test_access_collection_editor_page(self):
         self.assertIn('Introduction to Collections in Oppia', response.body)
         self.assert_can_edit(response.body)
         self.logout()
+
+    def test_collection_rights_handler(self):
+        collection_id = 'collection_id'
+        collection = collection_domain.Collection.create_default_collection(
+            collection_id, 'A title', 'A Category', 'An Objective')
+        collection_services.save_new_collection(self.owner_id, collection)
+
+        # Check that collections are published correctly
+        rights_manager.assign_role_for_collection(
+            self.owner_id, collection_id, self.editor_id,
+            rights_manager.ROLE_EDITOR)
+        rights_manager.publish_collection(self.owner_id, collection_id)
+
+        # Check that collections are published correctly
+        rights_manager.unpublish_collection(self.owner_id, collection_id)
```

Comment:
Yeah there's an exception being thrown at line 114. How do I check that if an error indeed arises? @seanlip